### PR TITLE
Test sign up

### PR DESCRIPTION
**Pull request by picollomartin**
# Summary 
- Add test for endpoint '/users' (POST)
- Add documentation of endpoint '/users'

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,9 +18,10 @@ services:
   - postgresql
 
 before_script:
-  - psql -c "CREATE DATABASE bookstest;" -U postgres
+  - psql -c "CREATE DATABASE travis;" -U postgres
```
Comment (gonziesc) [inline .travis.yml]: Why this change?

File: training-node/app/controllers/user.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+const userServices = require('../services/user'),
```
Comment (gonziesc) [inline training-node/app/controllers/user.js]: When you branch from something that has not been merged, you should compare to that branch and not to master in a Pull Request in order to see only the changes you made. You can change the base branch easily in the pr edition.

File: training-node/test/users.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    });
+    it('should fail creation because of missing parameter password', done => {
+      chai
+        .request(server)
+        .post('/users')
+        .send({ name: 'martin', lastName: 'picollo', email: 'martin2@wolox.com.ar' })
+        .catch(err => {
+          err.should.have.status(400);
+          err.response.should.be.json;
+          err.response.body.should.have.property('error');
```
Comment (gonziesc) [inline training-node/test/users.js:28]: Usually I also validate that the error message is correct, its no necessary for this training though

**Review by gonziesc — CHANGES_REQUESTED** [review]

File: training-node/test/users.js (hunk 3 above)
Comment (picollomartin) [reply training-node/test/users.js:28]: I'll keep it in mind :+1:

File: training-node/app/controllers/user.js (hunk 2 above)
Comment (picollomartin) [reply training-node/app/controllers/user.js]: Done !

**Review by gonziesc — APPROVED** [review]
